pages/media.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title> ORA UI </title>
+    <link href="../libs/font-awesome/css/font-awesome.css" rel="stylesheet">
+    <link href="../libs/Ionicons/css/ionicons.css" rel="stylesheet">
+    <link rel="stylesheet" href="../css/magen-iot-admin.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../css/oraui.css">
+</head>
+
+<body class="grey">
+
+    <!-- Menu sidebar -->
+    <button id="back" onclick="goTo('home.html')">
```

Comment:
instead of onclick="goTo('home.html')" put "class='backToMain'". the nav.js specifies that buttons with that class go back to home.html